projects/observability/src/shared/components/explore-query-editor/explore-query-editor.component.ts

```diff
@@ -32,27 +33,38 @@ import {
       </div>
 
       <div class="query-level-config">
-        <ht-explore-query-interval-editor
-          class="interval"
-          [interval]="currentVisualization.interval"
-          (intervalChange)="this.setInterval($event)"
-        ></ht-explore-query-interval-editor>
-        <ht-explore-query-group-by-editor
-          class="group-by"
-          [context]="currentVisualization.context"
-          [groupByExpression]="(currentVisualization.groupBy?.keyExpressions)[0]"
-          (groupByExpressionChange)="this.updateGroupByExpression(currentVisualization.groupBy, $event)"
-        ></ht-explore-query-group-by-editor>
-
-        <ht-explore-query-limit-editor
-          *ngIf="currentVisualization.groupBy"
-          class="limit"
-          [limit]="currentVisualization.groupBy?.limit"
-          (limitChange)="this.updateGroupByLimit(currentVisualization.groupBy!, $event)"
-          [includeRest]="currentVisualization.groupBy?.includeRest"
-          (includeRestChange)="this.updateGroupByIncludeRest(currentVisualization.groupBy!, $event)"
-        >
-        </ht-explore-query-limit-editor>
+        <div class="filters-row">
+          <ht-explore-query-interval-editor
+            class="interval"
+            [interval]="currentVisualization.interval"
+            (intervalChange)="this.setInterval($event)"
+          ></ht-explore-query-interval-editor>
+          <ht-explore-query-group-by-editor
+            class="group-by"
+            [context]="currentVisualization.context"
+            [groupByExpression]="(currentVisualization.groupBy?.keyExpressions)[0]"
+            (groupByExpressionChange)="this.updateGroupByExpression(currentVisualization.groupBy, $event)"
+          ></ht-explore-query-group-by-editor>
+
+          <ht-explore-query-limit-editor
+            *ngIf="currentVisualization.groupBy"
+            class="limit"
+            [limit]="currentVisualization.groupBy?.limit"
+            (limitChange)="this.updateGroupByLimit(currentVisualization.groupBy!, $event)"
+            [includeRest]="currentVisualization.groupBy?.includeRest"
+            (includeRestChange)="this.updateGroupByIncludeRest(currentVisualization.groupBy!, $event)"
+          >
+          </ht-explore-query-limit-editor>
+        </div>
+        <div class="filters-row" *htIfFeature="'${ApplicationFeature.ExplorerOrderBySupport}' | htFeature">
+          <ht-explore-query-order-by-editor
+            *ngIf="!currentVisualization.interval"
```

Comment:
If the interval is added back, the order by would be hidden - but would the state be cleared? We need to make sure we're not generating a query with both things.

Reply:
done